<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>slotsAutoDescribe</class>
 <widget class="QDialog" name="slotsAutoDescribe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 3px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 1px 0 1px;
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Slot Game Description Generator</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>No. of reels</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="reelNumSB">
          <property name="minimum">
           <number>3</number>
          </property>
          <property name="maximum">
           <number>7</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Reel Height</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="reelHieghtSB">
          <property name="minimum">
           <number>3</number>
          </property>
          <property name="maximum">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>No. of pay lines</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="payLineNumSB">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>No. of symbols (not wilds)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="symbolNumSB">
          <property name="minimum">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>No. of wild symbols</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="wildSymbolsSB">
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QCheckBox" name="wildsPayX">
          <property name="text">
           <string>Wilds pay</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="expandingWildsX">
          <property name="text">
           <string>Expanding wilds</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QCheckBox" name="transWildsX">
          <property name="text">
           <string>Transparent wilds</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QCheckBox" name="roamingWildsX">
          <property name="text">
           <string>Roaming wilds</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Pay line evaluation</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QComboBox" name="evaluationCB">
          <item>
           <property name="text">
            <string>Left To Right</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right To Left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Both Directions</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Unknown</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QCheckBox" name="stickyWildsX">
          <property name="text">
           <string>Sticky wilds</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>slotsAutoDescribe</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>slotsAutoDescribe</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
